src/templates/index.html

```diff
@@ -344,6 +356,230 @@ <h2 class="info-title">Paper Info:</h2>
+                Chrome Web Store</a>
+            </p>
+            <h2 id="explore-title">Explore your paper</h2>
+            <p>
+                Read the paper with an extra gear. After you upload a paper you can:
+            <ul>
+                <li>
+                    Navigate through pages
+                    <ul>
+                        <li>
+                            Click on the buttons <i class="material-icons">chevron_left</i><i
+                                class="material-icons">chevron_right</i>
+                        </li>
+                        <li>
+                            Scroll with the mouse
+                        </li>
+                        <li>
+                            Press on the arrow keys on the keyboard
+                        </li>
+                        <li>
+                            Or change page number from the input box
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Zoom in and out
+                    <ul>
+                        <li>
+                            Click on the buttons <i class="material-icons">zoom_in</i> <i
+                                class="material-icons">zoom_out</i>
+                        </li>
+                        <li>
+                            Press ctrl/cmd+ '+' or ctrl/cmd+ '-'
+                        </li>
+                        <li>
+                            Or change percentage from the input box
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Select text from the paper
+                </li>
+                <li>
+                    Reach page faster by clicking on the button &nbsp;<label>&#9776;</label>&nbsp;
+                    and scroll through thumbnails of the pages
+                </li>
+            </ul>
+            </p>
+            <h2 id="cross-references-title">Cross References</h2>
+            <p> Hover the references and extract data from them.
+            <ul>
+                <li>
+                    Finding a reference and hover it
+                    <ul>
+                        <li>
+                            If the reference is about an image you will see it in a small popup and a button
+                        </li>
+                        <li>
+                            If the reference is about a text you will see an extraction of it and a button
+                        </li>
+                        <li>
+                            If the reference is about a page you will see just the button
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    If you click the button, the page of the reference (entire page) will be open and you will have a
+                    two page layout
+                    You can also select text from it.
```

Comment:
Missing a period (.) between "layout" and "You"

Reply:
fixed